app/src/components/HomePage/TabHeader.tsx

```diff
@@ -60,12 +64,12 @@ export function TabHeader({
         {!isPublic ? (
           <Trans
             i18nKey="gpc-inventory-description"
-            values={{ year: year }}
+            values={{ year: inventory }}
```

Comment:
This seems wrong? Or is there a year number stored in a variable named `inventory`?